velox/exec/Task.h

```diff
@@ -125,6 +125,10 @@ class Task : public std::enable_shared_from_this<Task> {
 
   bool isUngroupedExecution() const;
 
+  /// Returns true if this task has ungrouped execution split under grouped
```

Comment:
This method may return different values during the lifetime of the task. It may return false before the ungrouped split arrives, then start returning true. This is probably OK, but it might not be obvious that method doesn't return same value all the time. Might be worth commenting.

Reply:
I changed to use numDriversUngrouped_ which is initialized by createDriverFactoriesLocked on task startup. I will make a comment there. Thanks!